packages/block/src/header.ts

```diff
@@ -616,8 +616,8 @@ export class BlockHeader {
   /**
    * Returns the signer address
    */
-  cliqueSignatureToAddress(): Address {
-    this._requireClique('cliqueSignatureToAddress')
+  cliqueSigner(): Address {
```

Comment:
Yeah, that's really a nice simplification, didn't look through that on first iteration.